### PR TITLE
Add configuration to allow transformation of durations to integers.

### PR DESCRIPTION
**Pull request by jeffmendoza**
The fluentd handler currently transforms durations to their string
representation for log messages. This commit adds a configuration to
override this default behaviour and tranform durations to integers of
unit millisecond.

### CODE REVIEW

File: mixer/adapter/fluentd/config/config.proto [hunk 1]
```diff
@@ -40,4 +40,9 @@ message Params {
   // Address of listening fluentd daemon. Example: fluentd-server:24224
   // Default value is localhost:24224
   string address = 1;
+
+  // Convert attributes of type duration to integers of unit ms in
+  // logs. Default behaviour is a string representation including
+  // unit.
+  bool integer_duration = 2;
```
Comment (jeffmendoza) [inline mixer/adapter/fluentd/config/config.proto]: I'm not sure if this should be an enum instead.

File: mixer/testdata/config/fluentd.yaml [hunk 2]
```diff
@@ -28,6 +28,7 @@ metadata:
   namespace: istio-system
 spec:
   address: "localhost:24224"
+  integerDuration: n
```
Comment (douglas-reid) [inline mixer/testdata/config/fluentd.yaml]: why 'n' here ?

File: mixer/adapter/fluentd/config/config.proto (hunk 1 above)
Comment (douglas-reid) [reply mixer/adapter/fluentd/config/config.proto]: enum would be more flexible, but I'm not quite sure what the other values would be.

should we have integer_duration be the default (and name this something like: `durations_as_string`) ?

File: mixer/testdata/config/fluentd.yaml (hunk 2 above)
Comment (jeffmendoza) [reply mixer/testdata/config/fluentd.yaml]: no, false

http://yaml.org/type/bool.html

File: mixer/adapter/fluentd/config/config.proto (hunk 1 above)
Comment (jeffmendoza) [reply mixer/adapter/fluentd/config/config.proto]: I don't mind switching the default, if that makes the most sense.

**Comment by douglas-reid** [comment]
let's rebase and push

**Comment by douglas-reid** [comment]
/lgtm

I'd still prefer ints as the default behavior, but since this has been around for awhile as-is without complaint, I think this is fine.

**Pull request merged**